Frontend/src/app/shared/ticker-range-picker/ticker-range-picker.component.scss

```
@@ -3,6 +3,12 @@
 
 :host {
   display: block;
+  // Establish a stacking context so the open dropdown renders above
```

Stylelint (17.9.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)